<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Draw_Window</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1179</width>
     <height>602</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="PaintBoard" name="painboard" native="true">
        <property name="minimumSize">
         <size>
          <width>1024</width>
          <height>512</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1024</width>
          <height>512</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:2px solid rgb(170, 0, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="func_label">
          <property name="minimumSize">
           <size>
            <width>1024</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1024</width>
            <height>35</height>
           </size>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="NoBrush">
               <color alpha="128">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="NoBrush">
               <color alpha="128">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="NoBrush">
               <color alpha="128">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);
border:2px solid rgb(0, 170, 255);
</string>
          </property>
          <property name="text">
           <string>公式为:None</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="res_label">
          <property name="minimumSize">
           <size>
            <width>1024</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1024</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);
border:2px solid rgb(0, 170, 255);</string>
          </property>
          <property name="text">
           <string>结果为:None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="pic_label">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:2px solid rgb(170, 0, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="rowWrapPolicy">
         <enum>QFormLayout::WrapLongRows</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="Next_button">
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="clean_button">
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>清空绘板</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="use_model_button">
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>识别</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="eraser_check">
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>橡皮</string>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="autoRepeatInterval">
           <number>100</number>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="eraser_label">
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>橡皮大小</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QSpinBox" name="eraser_spinbox">
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>80</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>130</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintBoard</class>
   <extends>QWidget</extends>
   <header>PaintBoard</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>clean_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>clean_draw()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1150</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>1236</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_model_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>use_model()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1079</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>1207</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eraser_check</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>eraser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1136</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>1199</x>
     <y>547</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eraser_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>change_eraser_size()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1104</x>
     <y>593</y>
    </hint>
    <hint type="destinationlabel">
     <x>1066</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Next_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>next_pic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1096</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>1190</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clean_draw()</slot>
  <slot>use_model()</slot>
  <slot>eraser()</slot>
  <slot>change_eraser_size()</slot>
  <slot>next_pic()</slot>
 </slots>
</ui>
